src/containers/Application/ApplicationHome.tsx

```diff
@@ -45,6 +45,12 @@ const ApplicationHome: React.FC<ApplicationProps> = ({ structure, template }) =>
 
   if (!fullStructure || !fullStructure.responsesByCode) return <Loading />
 
+
+  const canUserEdit = () => {
+    const { status } = fullStructure.info.current as StageAndStatus
```

Comment:
It seem easier to just do fullStructure.info?.current === ApplicationStatus.Draft (or is there a problem with types ?).

Anyways this is stylistic comment no need to change

Reply:
Done: edc427a